categories.html

```diff
@@ -0,0 +1,62 @@
+---
+layout: page
+image: /images/posts.jpg
+permalink: /categories/
+title: Categories
+---
+
+
+
+{% assign date_format = site.minima.date_format | default: "%b %-d, %Y" %}
+<div id="archives">
+{% for category in site.categories %}
+  <div class="archive-group">
+    {% capture category_name %}{{ category | first }}{% endcapture %}
+    <div id="#{{ category_name | slugize }}"></div>
+      <p></p>
+
+      <h3 class="category-head">{{ category_name }}</h3>
+      <a name="{{ category_name | slugize }}"></a>
+      {% for post in site.categories[category_name] %}
+    <article class="archive-item" >
+      
+      <div class="post">
```

Comment:
To avoid redundancy and keep things synchronized, you could move the part to create the post card to a dedicated HTML file in the `_includes` folder and call it from here and also in the `posts.html` file

Reply:
i couldn't because i dont need the whole card, i excluded the tag and the content [just image and title]
and somehow when calling from post.html into categories.html, it just replicates the stories page without regards for the categories function.